Change various example paths in the config files to match SUSE.

Index: xen-unstable/tools/examples/xmexample1
===================================================================
--- xen-unstable.orig/tools/examples/xmexample1
+++ xen-unstable/tools/examples/xmexample1
@@ -7,11 +7,14 @@
 #============================================================================
 
 #----------------------------------------------------------------------------
-# Kernel image file.
-kernel = "/boot/vmlinuz-2.6.10-xenU"
-
-# Optional ramdisk.
-#ramdisk = "/boot/initrd.gz"
+# Kernel image file and (optional) ramdisk (initrd).
+kernel = "/boot/vmlinuz-xen"
+ramdisk = "/boot/initrd-xen"
+
+# Or use domUloader instead of kernel/ramdisk to get kernel from domU FS
+#bootloader = "/usr/lib/xen/boot/domUlaoder.py"
+#bootentry = "hda2:/vmlinuz-xen,/initrd-xen"
+#bootentry = "/boot/vmlinuz-xen,/boot/initrd-xen"
 
 # The domain build function. Default is 'linux'.
 #builder='linux'
@@ -96,7 +99,7 @@ disk = [ 'phy:hda1,hda1,w' ]
 #hostname= "vm%d" % vmid
 
 # Set root device.
-root = "/dev/hda1 ro"
+root = "/dev/hda1"
 
 # Root device for nfs.
 #root = "/dev/nfs"
@@ -105,8 +108,8 @@ root = "/dev/hda1 ro"
 # Root directory on the nfs server.
 #nfs_root   = '/full/path/to/root/directory'
 
-# Sets runlevel 4.
-extra = "4"
+# Extra arguments to pass to the kernel.
+extra = ""
 
 #----------------------------------------------------------------------------
 # Configure the behaviour when a domain exits.  There are three 'reasons'
Index: xen-unstable/tools/examples/xmexample2
===================================================================
--- xen-unstable.orig/tools/examples/xmexample2
+++ xen-unstable/tools/examples/xmexample2
@@ -35,11 +35,14 @@ xm_vars.var('vmid',
 xm_vars.check()
 
 #----------------------------------------------------------------------------
-# Kernel image file.
-kernel = "/boot/vmlinuz-2.6.10-xenU"
-
-# Optional ramdisk.
-#ramdisk = "/boot/initrd.gz"
+# Kernel image file and (optional) ramdisk (initrd).
+kernel = "/boot/vmlinuz-xen"
+ramdisk = "/boot/initrd-xen"
+
+# Or use domUloader instead of kernel/ramdisk to get kernel from domU FS
+#bootloader = "/usr/lib/xen/boot/domUlaoder.py"
+#bootentry = "hda2:/vmlinuz-xen,/initrd-xen"
+#bootentry = "/boot/vmlinuz-xen,/boot/initrd-xen"
 
 # The domain build function. Default is 'linux'.
 #builder='linux'
@@ -132,7 +135,7 @@ disk = [ 'phy:sda%d,sda1,w' % (7+vmid),
 #hostname= "vm%d" % vmid
 
 # Set root device.
-root = "/dev/sda1 ro"
+root = "/dev/sda1"
 
 # Root device for nfs.
 #root = "/dev/nfs"
@@ -141,8 +144,8 @@ root = "/dev/sda1 ro"
 # Root directory on the nfs server.
 #nfs_root   = '/full/path/to/root/directory'
 
-# Sets runlevel 4 and the device for /usr.
-extra = "4 VMID=%d usr=/dev/sda6" % vmid
+# Sets the device for /usr.
+extra = "VMID=%d usr=/dev/sda6" % vmid
 
 #----------------------------------------------------------------------------
 # Configure the behaviour when a domain exits.  There are three 'reasons'
Index: xen-unstable/tools/examples/xmexample3
===================================================================
--- xen-unstable.orig/tools/examples/xmexample3
+++ xen-unstable/tools/examples/xmexample3
@@ -35,11 +35,14 @@ xm_vars.var('vmid',
 xm_vars.check()
 
 #----------------------------------------------------------------------------
-# Kernel image file.
-kernel = "/path/to/domU/kernel"
-
-# Optional ramdisk.
-#ramdisk = "/boot/initrd.gz"
+# Kernel image file and (optional) ramdisk (initrd).
+kernel = "/boot/vmlinuz-xen"
+ramdisk = "/boot/initrd-xen"
+
+# Or use domUloader instead of kernel/ramdisk to get kernel from domU FS
+#bootloader = "/usr/lib/xen/boot/domUlaoder.py"
+#bootentry = "hda2:/vmlinuz-xen,/initrd-xen"
+#bootentry = "/boot/vmlinuz-xen,/boot/initrd-xen"
 
 # The domain build function. Default is 'linux'.
 #builder='linux'
Index: xen-unstable/tools/examples/xmexample.hvm
===================================================================
--- xen-unstable.orig/tools/examples/xmexample.hvm
+++ xen-unstable/tools/examples/xmexample.hvm
@@ -70,7 +70,7 @@ vif = [ 'type=ioemu, bridge=xenbr0' ]
 # and MODE is r for read-only, w for read-write.
 
 #disk = [ 'phy:hda1,hda1,r' ]
-disk = [ 'file:/var/images/min-el3-i386.img,hda,w', ',hdc:cdrom,r' ]
+disk = [ 'file:/var/lib/xen/images/disk.img,ioemu:hda,w', ',hdc:cdrom,r' ]
 
 #----------------------------------------------------------------------------
 # Configure the behaviour when a domain exits.  There are three 'reasons'
Index: xen-unstable/docs/man/xmdomain.cfg.pod.5
===================================================================
--- xen-unstable.orig/docs/man/xmdomain.cfg.pod.5
+++ xen-unstable/docs/man/xmdomain.cfg.pod.5
@@ -38,13 +38,13 @@ file.  
 
 The kernel image for the domain.  The format of the parameter is the
 fully qualified path to the kernel image file,
-i.e. I</boot/vmlinuz-2.6.12-xenU>.
+i.e. I</boot/vmlinuz-xen>.
 
 
 =item B<ramdisk>
 
 The initial ramdisk for the domain.  The format of the parameter is
-the fully qualified path to the initrd, i.e. I</boot/initrd.gz>.  On
+the fully qualified path to the initrd, i.e. I</boot/initrd-xen>.  On
 many Linux distros you will not need a ramdisk if using the default
 xen kernel.
 
@@ -258,14 +258,14 @@ configured.  They should not be consider
 
 =item I<A Loopback File as Root>
 
-    kernel = "/boot/vmlinuz-2.6-xenU"
+    kernel = "/boot/vmlinuz-xen"
     memory = 128
     name = "MyLinux"      
-    root = "/dev/hda1 ro"
-    disk = [ "file:/var/xen/mylinux.img,hda1,w" ]
+    root = "/dev/hda1"
+    disk = [ "file:/var/lib/xen/images/MyLinux/hda1,hda1,w" ]
 
 This creates a domain called MyLinux with 128 MB of memory using a
-default xen kernel, and the file /var/xen/mylinux.img loopback mounted
+default xen kernel, and the file hda1 loopback mounted
 at hda1, which is the root filesystem.
 
 =item I<NFS Root>
Index: xen-unstable/docs/man/xm.pod.1
===================================================================
--- xen-unstable.orig/docs/man/xm.pod.1
+++ xen-unstable/docs/man/xm.pod.1
@@ -102,8 +102,8 @@ soon as it is run.
 
 =item I<without config file>
  
-  xm create /dev/null ramdisk=initrd.img \
-     kernel=/boot/vmlinuz-2.6.12.6-xenU \
+  xm create /dev/null ramdisk=initrd-xen \
+     kernel=/boot/vmlinuz-xen \
      name=ramdisk nics=0 vcpus=1 \
      memory=64 root=/dev/ram0
 
